Automattic/jetpack
# Masterbar: Add Calypso support to Testimonials and Portfolios

Points Testimonial and Portfolio post types to Calypso. This was updated in the link destination spreadsheet after I started working on the original menu structure code. At this point we only handle these two CPTs in Calypso.

Fixes #18304

#### Changes proposed in this Pull Request:

* Adds a general CPT menu method, covered by tests
* Adds to post type specific methods for Testimonial and Portfolio post types.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Apply D55179-code to your sandbox and [activate the Dara theme](https://wordpress.com/theme/dara).
* Switch between Calypso and wp-admin and make sure the order of the menu items stay the same and they menu items link to Calypso
* Any taxonomy menu items should still point to wp-admin.

#### Proposed changelog entry for your changes:


None needed.
